### PR TITLE
Add support for highlighted items [issue-10]

### PR DESCRIPTION
**Pull request by JaySunSyn**
See issue https://github.com/Neovici/cosmoz-omnitable/issues/10

Please merge https://github.com/Neovici/cosmoz-grouped-list/pull/9 first!

### CODE REVIEW

**Comment by JaySunSyn** [comment]
@nomego In this PR I do not consider questions like:
- can a selected row get highlighted?
- should a highlighted row that gets selected still be highlighted?

**Comment by nomego** [comment]
Great, but I think we should be able to configure the behavior if it's multi or single highlight with a property.

**Comment by JaySunSyn** [comment]
@nomego what about checking if the input is an array or an object and deciding on multi/single based on that?

**Comment by JaySunSyn** [comment]
@nomego please check

File: cosmoz-omnitable.html [hunk 1]
```diff
@@ -173,15 +173,16 @@
...
 						<cosmoz-grouped-list-template id="itemTemplate">
 							<template>
 								<div class="item-row-wrapper">
-									<div class$="[[ _getItemRowClasses(selected) ]]">
+									<div selected$="[[selected]]" class="itemRow" highlighted$="[[highlighted]]">
 										<div class="selectItemCheckbox" hidden$="[[ !_showCheckboxes ]]">
 											<paper-checkbox checked="{{ selected }}" on-change="_onItemCheckboxChange"></paper-checkbox>
 										</div>
 										<cosmoz-omnitable-item-row
+											on-tap="_itemRowTapped"
```
Comment (plequang) [inline cosmoz-omnitable.html]: You can remove that event handler for now.

**Pull request merged**